go.mod

```
@@ -184,7 +183,10 @@ require (
 	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 )
 
-require cloud.google.com/go/compute/metadata v0.2.3
+require (
+	cloud.google.com/go/compute/metadata v0.2.3
+	github.com/alecthomas/kingpin/v2 v2.3.1
```

Please add this comment and make sure it survives any changes/tidying:

```suggestion
	github.com/alecthomas/kingpin/v2 v2.3.1 // replaced
```

This is useful to draw attention to bots trying to update forked dependencies.